components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -0,0 +1,158 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@class MDCActionSheetAction;
+
+@interface MDCActionSheetController : UIViewController
+
+
+/**
+ Convenience constructor to create and return a view controller for displaying an alert to the user.
+
+ After creating the alert controller, add actions to the controller by calling -addAction.
+
+ @param title The title of the alert.
+ @return An initialized MDCActionSheetController object.
+ */
++ (nonnull instancetype)actionSheetControllerWithTitle:(nullable NSString *)title;
+
+/**
+ Convenience constructor to create and return a view controller for displaying an alert to the user.
+
+ After creating the alert controller, add actions to the controller by calling -addAction.
+
+ @param title The title of the alert.
+ @param message Descriptive text that summarizes a decision in a sentence of two.
+ @return An initialized MDCActionSheetController object.
+ */
++ (nonnull instancetype)actionSheetControllerWithTitle:(nullable NSString *)title
+                                               message:(nullable NSString *)message;
+
+- (nonnull instancetype)initWithNibName:(NSString *)nibNameOrNil
+                                 bundle:(NSBundle *)nibBundleOrNil;
+
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder;
```

Comment:
I believe you don't have to redefine these APIs because they're part of the parent class, should be able to remove.

Reply:
Done

Reply:
I would mark initWithCoder: and initWithNibName:bundle NS_UNAVAILABLE.  I don't expect we would support either.